demo-app/package.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": {
     "final-form": "^4.20.1",
     "js-cookie": "^2.2.1",
-    "next": "10.0.3",
+    "next": "12.1.0",
```

Comment:
why is next.js in deps?

Reply:
Oops, forgot that it was part of Demo app json. Ignore me. Also did we check if that demo app actually works?

Reply:
I just ran the demo app. Seems to work fine.